src/components/RiskComponents.vue

```diff
@@ -1,84 +1,162 @@
 <template>
   <v-container v-if="hasRiskData">
-    <v-row v-if="onMediumandUp">
-      <v-col cols="12" md="3" class="crowding">
-        <RiskComponent :activity="activity" type="crowding"></RiskComponent>
-      </v-col>
-      <v-col cols="12" md="3" class="droplets" :class="maybeSeparator">
-        <RiskComponent :activity="activity" type="droplets"></RiskComponent>
-      </v-col>
-      <v-col cols="12" md="3" class="exposureTime" :class="maybeSeparator">
-        <RiskComponent :activity="activity" type="exposureTime"></RiskComponent>
-      </v-col>
-      <v-col cols="12" md="3" class="ventilation" :class="maybeSeparator">
-        <RiskComponent :activity="activity" type="ventilation"></RiskComponent>
+    <!-- displays on medium and up breakpoints -->
+    <v-row class="hidden-sm-and-down">
+      <v-col
+        v-for="(risk, index) in riskData"
+        :key="risk.type"
+        cols="3"
+        :class="index && 'left-border'"
```

Comment:
As another idea (though this made sense as is to me), these computed classes have a few styles -- I think there is one where a computed value or a method can return either the class name or a boolean to enable the class, and in there you could do `index > 0` instead of depending on the falsyness of `0`.

Something like `:class="{ 'left-border': useLeftBorder(index) }"`